getting-started.md

```diff
@@ -21,14 +21,39 @@ To log in to the Application Service Adapter with the cf CLI:
 
     > **Note:**  If you configured the Application Service Adapter with a globally trusted certificate during installation, you can omit the `--skip-ssl-validation` flag.
 
-1. Log in with the cf CLI.
+2. Log in with the cf CLI.
 
     ```bash
     cf login
     ```
 
     The cf CLI takes you through an interactive flow to set your user name, and detects the user accounts in your local Kubeconfig file. Select one that is a cluster-admin for the Kubernetes cluster to which the Application Service Adapter is installed.
```

Comment:
Does the CF admin need to be a cluster admin too, or could they be a separate, "normal" user that then gets assigned this admin role?